cloudformation.yml

```diff
@@ -48,6 +48,14 @@ Parameters:
     MinValue: 1
     Default: 25
 
+  SystemAvailable:
+    Description: Set to false to shutdown system, API will run and provide errors to users, but will not accept jobs.
```

Comment:
I wonder if this should be called out specifically in the readme, or if the note to review parameters is sufficient.

If I wasn't familiar with this app but needed to shut it down, this might be hard to find.